eng/Publishing.props

```diff
@@ -70,7 +71,7 @@
     </ItemGroup>
 
     <!-- Create Artifact items for produced packages and assets -->
-    <ItemGroup>
+    <ItemGroup Condition="'$(IsPublishPass2)' == 'false'">
```

Comment:
Check on `!= true` as in eng/PublishSourceBuild.props? That makes the property a true boolean instead of checking on true, false and other values.